design-tokens/tokens/primitive/primitives.default.json

```diff
@@ -802,6 +802,18 @@
             }
           }
         },
+        "450": {
```

Comment:
FYI -- Adding this color now to match Joelle's theme update designs. We have a lot of similar greens, so it would be worth revisiting and seeing if any can be pruned/consolidated.